tests/_data/schema.sql

```diff
@@ -81,4 +84,113 @@ CREATE TABLE `user` (
   UNIQUE KEY `user_unique_username` (`username`)
 ) ENGINE=InnoDB AUTO_INCREMENT=7 DEFAULT CHARSET=utf8;
 
+
+DROP TABLE IF EXISTS `auth_assignment`;
```

Comment:
why you added RBAC tables to the schema? usuario uses yii's core rbac tables and migration

Reply:
To use `$I->amLoggedInAs()`

Reply:
@Eseperio it means that Usuario already uses RBAC`s tables from Yii.

Reply:
This is the schema only for tests, not migrations.